src/components/animations/CheckmarkAnimation.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { View } from 'react-native';
+import RadialGradient from 'react-native-radial-gradient';
+import Animated, {
+  Easing,
+  useAnimatedStyle,
+  useDerivedValue,
+  withSequence,
+  withTiming,
+} from 'react-native-reanimated';
+import { Path, Svg } from 'react-native-svg';
+import { Flex } from '../layout';
+import styled from '@rainbow-me/styled-components';
+
+const Container = styled(Flex)({
+  justifyContent: 'center',
+  width: '100%',
+});
+
+const Circle = styled(Flex)({
+  alignItems: 'center',
+  borderRadius: 120,
+  height: 120,
+  justifyContent: 'center',
+  overflow: 'hidden',
+  width: 120,
+});
+
+const AnimatedView = Animated.createAnimatedComponent(View);
```

Comment:
just use `Animated.View`, https://docs.swmansion.com/react-native-reanimated/docs/fundamentals/animations

Reply:
Resolved in #3158